src/VisualizationTests/PerformanceVisualizationTests.cs

```diff
@@ -18,14 +18,16 @@ namespace WpfVisualizationTests
     public class PerformanceVisualizationTests : VisualizationTest
     {
         // Determines how many times each performance graph will be run.
-        const int SamplingAmount = 10;
+        const int SamplingAmount = 3;
 
         [Test]
         [TestCaseSource("PerformanceTestSource")]
         public void TestVisualizationPerformance(string filePath)
         {
             ViewModel.OpenCommand.Execute(filePath);
+            DispatcherUtil.DoEvents();
             RunCurrentModel();
+            DispatcherUtil.DoEvents();
```

Comment:
Do these tests now run as part of the other performance benchmark tests? It's been a long time since I ran those tests locally. It would be interesting to see what are the render time results from running these.

Reply:
These tests relation with performance benchmark tests is only that by default they use the same graphs. These are aimed to just measure rendering time, which is something that gets mixed with a lot of other stuff in the performance benchmark.
Also these tests are not really a tool, you need to do some tweaking and do some extra steps to get results out of them.

Reply:
Can you include comments on how to run these tests and see the timing results?

Reply:
It's in the top of this file, in the class comment.

Reply:
`For that, you should make sure the renderTimer Stopwatch in HelixWatch3DViewModel is enabled and logging to Console. Output should be redirected to a file.`

Does one need to add a `TextWriterTraceListener` for this? Can you give an example of how would someone carry out the steps above?

Reply:
I haven't explored using `TextWriterTraceListener`. For my testing I just run the test using nunit from the command line and redirect output to a file using `>`.
If you want to run in `Release` mode, you need to do changes to make sure the Stopwatch is enabled. Removing the `#if DEBUG` surrounding the timer where it's defined and used should be needed. Also, to guarantee you are logging to Console in Release, you should use `Console.WriteLine` rather than `Debug.WriteLine`.

Reply:
Thanks. Should we just change `Debug.WriteLine` to `Console.WriteLine` in that case?

Reply:
I think `Debug.WriteLine` should be considered a better practice in general, even if under a `#if DEBUG` flag. The possibility to add trace listeners is there, even if we don't use it today. In general we are using `Debug.WriteLine` in other places to log stuff, so I think it's more consistent if we keep it that way.